site/docs/guide/introduction.md

```diff
@@ -129,9 +130,46 @@ Please choose what you think is best for you.
 
 ### Prerequisites for Deno
 
-[Install Deno](https://deno.land/#installation) if you have not done that already.
-When you have created your bot, for example in a file called `bot.ts`, you can run it via `deno run --allow-net bot.ts`.
-You can stop it again with `Ctrl+C`.
+Before you can start creating your bot, let's first spend a few minutes on a proper setup for software development.
+This means installing a few tools.
+
+#### Preparing Your Machine for Development
+
+[Install Deno](https://deno.land/manual/getting_started/installation#download-and-install) if you have not done that already.
+
+You also need a text editor that works well for coding.
+The best one for Deno is Visual Studio Code.
+[Install it.](https://code.visualstudio.com/)
+
+Next, you need to connect VSCode and Deno.
```

Comment:
```suggestion
Next, you need to connect Visual Studio Code and Deno.
```

Reply:
We are already using VSCode in more places in the docs and I don't mind. It's the more common term and it's more readable, too. I find this a good place to implicitly introduce this.

Do you maybe want to introduce the term explicitly instead, assuming that people don't get what we're talking about?

Reply:
But is it VSCode or VS Code?

Reply:
Notice https://github.com/microsoft/vscode#related-projects

Reply:
We can do VS Code, but we'll have to do it consistently everywhere. We currently use VSCode at least once.

Reply:
OK. In another PQ.